ParisBNF/abb/BNFabb87.xml

```diff
@@ -0,0 +1,214 @@
+                            <locus from="140r" to="148v"/>
+                            <title type="complete"><foreign xml:lang="gez"> ʾaryām antiphons</foreign></title>
+                        </msItem>
+</msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="149r" to="173v"/>
+                            <title type="complete" ref="LIT2610Zemmar">Zǝmmāre 
+                            </title>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="175r" to="195v"/>
+                            <title type="complete" ref="LIT3186Meeraf">Mǝʿrāf, Common of the Office 
+                                
+                            </title>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="195v" to="209r"/>
+                            <title type="complete" ref="LIT1990Mawase">Mawāśǝʾt, Antiphonary for feasts and funeral service
+                            </title>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="60">209</measure>
+                                    <dimensions type="outer"/>
+                                    <height unit="mm">375</height>
+                                    <width unit="mm">320</width>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="46">   
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg> 
+                                <desc><persName
+                                    ref="PRS1071dAbbadi">A. d'Abbadie</persName> describes the handwriting as <foreign xml:lang="fr">'fine et jolie'</foreign>.    
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#209v" ></locus>
```

Comment:
Remove space in `<locus target="#209v" >`. I think a space inside a xml-tag can cause problems in the app.